plugin-server/src/main/ingestion-queues/session-recording/services/replay-events-ingester.ts

```diff
@@ -138,6 +138,11 @@ export class ReplayEventsIngester {
                 if (replayRecord !== null) {
                     const asDate = DateTime.fromSQL(replayRecord.first_timestamp)
                     if (!asDate.isValid || Math.abs(asDate.diffNow('day').days) >= 7) {
+                        const eventTypes = rrwebEvents.map((event) => ({
```

Comment:
Two loops over what could potentially be arrays with 1000s of entries? Given how sensitive blobby is to cpu work, maybe we could:
1. Do this in one loop rather than a map and filter
2. Wait till after christmas to merge

Reply:
1. fair
2. definitely!